src/openklant/conf/dev.py

```diff
@@ -15,7 +15,6 @@
 
 os.environ.setdefault("ENVIRONMENT", "development")
 os.environ.setdefault("DISABLE_2FA", "yes")
-os.environ.setdefault("LOG_FORMAT_CONSOLE", "plain_console")
```

Comment:
I don't think we want to remove this, let's keep the default `plain_console` for dev settings